packages/ripgrep-workspace/src/node/ripgrep-workspace.spec.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2017 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as chai from 'chai';
+import * as stream from 'stream';
+import { testContainer } from './test/inversify.spec-config';
+import { RipGrepProcessFactory, RipGrepWorkSpace } from './ripgrep-workspace';
+
+const expect = chai.expect;
+let ripgrepWorkSpace: RipGrepWorkSpace;
+const ripgrepProcessFactory = testContainer.get<RipGrepProcessFactory>(RipGrepProcessFactory);
+
+describe("ripgrep-workspace", () => {
+    it('test basic option', (done) => {
+
+        ripgrepWorkSpace = ripgrepProcessFactory({ 'args': ['--version'] });
+        const outStream = new stream.PassThrough();
+
+        const p = new Promise<String>((resolve, reject) => {
+            outStream.on('data', data => {
+                resolve(data.toString());
+            });
+        });
+        ripgrepWorkSpace.output.pipe(outStream);
+        p.then((data) => {
+            expect(data).to.include("ripgrep");
+            done();
+        })
+    });
+    it('test basic grep', (done) => {
+        {
+            ripgrepWorkSpace = ripgrepProcessFactory({ 'args': ['theia *'] });
+            const outStream = new stream.PassThrough();
+
+            const p = new Promise<String>((resolve, reject) => {
+                outStream.on('data', data => {
+                    resolve(data.toString());
+                });
+            });
+            ripgrepWorkSpace.output.pipe(outStream);
+            p.then((data) => {
+                expect(data).to.include("theia");
+                done();
+            })
```

Comment:
You just actually write that as
`return expect(p).to.eventually.include("theia")`
same thing for all the tests